### ISSUE TITLE
Problems on install

### ISSUE THREAD

**Issue by rafaell-lycan** [munkyjunky/generator-hubot-script-babel#1]
On run `yo hubot-script-babel`, this error happened:

``` 
Error: ENOENT: no such file or directory, open '/usr/local/lib/node_modules/generator-hubot-script-babel/generators/app/templates/.gitignore'
    at Error (native)
    at Object.fs.openSync (fs.js:585:18)
    at Function.module.exports.sync (/usr/local/lib/node_modules/generator-hubot-script-babel/node_modules/read-chunk/index.js:34:14)
    at copy (/usr/local/lib/node_modules/generator-hubot-script-babel/node_modules/yeoman-generator/lib/actions/actions.js:38:27)
    at /usr/local/lib/node_modules/generator-hubot-script-babel/generators/app/index.js:62:10
    at process._tickCallback (node.js:382:9)
```

**Comment by munkyjunky** [comment]
Thanks for raising - looks like the nested .npmignore is stopping a few files being published. Will get this sorted ASAP.

**Comment by munkyjunky** [comment]
This issue should be resolved in version 0.6.1. Thanks for raising this!
